lib/widgets/action_sheet.dart

```diff
@@ -69,11 +98,21 @@ abstract class MessageActionSheetMenuItemButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final designVariables = DesignVariables.of(context);
     final zulipLocalizations = ZulipLocalizations.of(context);
     return MenuItemButton(
-      leadingIcon: Icon(icon),
+      trailingIcon: Icon(icon, color: designVariables.actionSheetMenuButtonForeground),
+      style: MenuItemButton.styleFrom(
+        padding: const EdgeInsets.symmetric(vertical: 12, horizontal: 16),
+        tapTargetSize: MaterialTapTargetSize.shrinkWrap,
+        minimumSize: Size.zero,
+        backgroundColor: designVariables.actionSheetMenuButtonBackground,
+        foregroundColor: designVariables.actionSheetMenuButtonForeground,
+      ),
       onPressed: () => onPressed(context),
-      child: Text(label(zulipLocalizations)));
+      child: Text(label(zulipLocalizations),
+        style: const TextStyle(fontSize: 20, fontWeight: FontWeight.w600),
```

Comment:
I think fontWeight is not being applied correctly, you need to use `weightVariableTextStyle`:
```suggestion
        style: const TextStyle(fontSize: 20)
          .merge(weightVariableTextStyle(context, wght: 600)),
```

Reply:
Similarly for the Cancel button.